pallets/creditcoin/src/benchmarking.rs

```diff
@@ -318,6 +317,11 @@ benchmarks! {
 		let task_output = crate::TaskOutput::from((collected_coins_id, collected_coins));
 	}: persist_task_output(RawOrigin::Signed(authority), deadline, task_output)
 
+	del_authority{
+		let root = RawOrigin::Root;
+		let who = authority_account::<T>(false);
+		<Creditcoin<T>>::add_authority(root.clone().into(), who.clone());
```

Comment:
```suggestion
let _result = <Creditcoin<T>>::add_authority(root.clone().into(), who.clone());
```

to make Clippy happy.